src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -32,13 +64,75 @@ public DistributedApplication(IHost host)
     /// Creates a new instance of the <see cref="IDistributedApplicationBuilder"/> interface.
     /// </summary>
     /// <returns>A new instance of the <see cref="IDistributedApplicationBuilder"/> interface.</returns>
+    /// <remarks>
+    /// This overload of the <see cref="CreateBuilder()"/> method should only be
+    /// used when the AppHost is not intended to be used wtih a deployment tool. Because no arguemnts are
```

Comment:
`AppHost`, is it a concept, a type, a noun? The usage of pascal casing and no documentation tags is confusing.

Reply:
See @IEvangelist comments below about what they are doing in learn.ms.com docs. We'll switch to that pattern for consistency.